core/integration/container_test.go

```diff
@@ -3496,14 +3496,14 @@ func TestContainerWithMountedCacheOwner(t *testing.T) {
 	t.Run("permissions (source)", func(t *testing.T) {
 		dir := c.Directory().
 			WithNewDirectory("perms", dagger.DirectoryWithNewDirectoryOpts{
-				Permissions: 0745,
+				Permissions: 0o745,
```

Comment:
Is this coming from a super opinionated formatter? Mostly curious, in the wild I usually see octal notation without the 'o'.

Reply:
it's coming from gofumpt https://github.com/mvdan/gofumpt
![image](https://github.com/dagger/dagger/assets/1578458/27aece39-967e-4472-bb14-8c0010f78cb6)

IIRC we already have some other octals in the code with this format that's why I didn't change it. I can rollback this change if necessary. 

Did a quick search Go's codebase and seems like they don't use it consistently either: https://github.com/golang/go/blob/862fa6d099fb046e90efd537b2c0ac2667c23d90/src/cmd/go/go_test.go#L264

Reply:
Eh it's fine as-is, maybe the world is trending towards the new syntax. :man_shrugging: Thanks for looking into it!